server/web/models.py

```diff
@@ -0,0 +1,30 @@
+from . import db
+from werkzeug.security import generate_password_hash, check_password_hash
+
+
+class Account(db.Model):
+    __tablename__ = 'account'
+    id = db.Column(db.Integer, primary_key=True)
+    companyName = db.Column(db.String(64), unique=True)
```

Comment:
use snake case `company_name`. This attributes get mapped to the column names in the database